features/install-profiles/src/test/java/org/codice/ddf/features/profiles/test/ITInstallProfilesFeatures.java

```diff
@@ -0,0 +1,69 @@
+package org.codice.ddf.features.profiles.test;
+
+import static org.codice.ddf.test.common.options.DebugOptions.defaultDebuggingOptions;
+import static org.codice.ddf.test.common.options.DistributionOptions.kernelDistributionOption;
+import static org.codice.ddf.test.common.options.FeatureOptions.addBootFeature;
+import static org.codice.ddf.test.common.options.FeatureOptions.addFeatureRepo;
+import static org.codice.ddf.test.common.options.LoggingOptions.defaultLogging;
+import static org.codice.ddf.test.common.options.PortOptions.defaultPortsOptions;
+import static org.codice.ddf.test.common.options.TestResourcesOptions.getTestResource;
+import static org.codice.ddf.test.common.options.TestResourcesOptions.includeTestResources;
+import static org.codice.ddf.test.common.options.VmOptions.defaultVmOptions;
+import static org.ops4j.pax.exam.CoreOptions.options;
+
+import java.util.Arrays;
+import java.util.List;
+import javax.inject.Inject;
+import org.apache.karaf.features.FeaturesService;
+import org.codice.ddf.test.common.features.FeatureUtilities;
+import org.codice.ddf.test.common.features.TestUtilitiesFeatures;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExamParameterized;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+
+@RunWith(PaxExamParameterized.class)
+@ExamReactorStrategy(PerClass.class)
+public class ITInstallProfilesFeatures {
+
+  private static final String FEATURE_REPO_PATH = getTestResource("/features.xml");
+
+  // TODO: tbatie - 4/9/18 - Make follow up ticket for features in profile-development not starting
+  // up.
+  private static final List<String> IGNORED_FEATURES = Arrays.asList("profile-development");
+
+  @Configuration
+  public static Option[] examConfiguration() {
```

Comment:
The main reason I didn't put these in some sort of `defaultExamOptions` method or create a base class to share between these tests is to avoid making the test environment configuration unclear. I hope we can strike a balance between making a readable test environment configuration and code duplication as we write more of these tests.